static/js/modules/game/countdown.js

```diff
@@ -64,7 +76,7 @@ var CountdownTimer = {
             <slot></slot>
             <p>Good Luck!</p>
 
-            <div><button id="start-btn" class="btn btn-outline-secondary">(Don't want to wait? Start immediately!)</button></div>
+            <div><button id="start-btn" class="btn btn-outline-secondary">(Don't want to wait? Click here or press spacebar start immediately!)</button></div>
```

Comment:
This is getting kind of wordy. Maybe just change to `Click here or press spacebar to start immediately!`